proposals/0008-swift-prometheus.md

```diff
@@ -0,0 +1,150 @@
+# SwiftPrometheus - Prometheus Metrics in Swift
+
+* Proposal: SSWG-0008
+* Authors: [Jari Koopman / LotU](https://github.com/MrLotU)
+* Review Manager: TBD
+* Status:  [Implemented ](https://github.com/MrLotU/SwiftPrometheus/tree/MetricsLib)
+* Pitch: [Pitches/Prometheus](https://forums.swift.org/t/client-side-prometheus-implementation/18098/)
+
+## Package Description
+Prometheus client side implementation.
+
+|  |  |
+|--|--|
+| **Package Name** | `SwiftPrometheus` |
+| **Module Name** | `Prometheus` & `PrometheusMetrics` |
```

Comment:
Prometheus -> PrometheusClient?

Reply:
Will update this. Thanks 👍

Reply:
Oh, misread. For the same reasons I mentioned above, I prefer not to rename to `PrometheusClient` (it's the name of the class in the module, not the module itself) but I will remove the `PrometheusMetrics` section since it's not there anymore :)